styles.css

```diff
@@ -640,11 +662,28 @@ header p {
     align-items: center;
   }
 
+  .card .first-child {
+    width: 100%;
+  }
+
+  .card .wallpaper {
+    width: 100%;
+    height: 486px;
+  }
+
   .card .card-part2 {
     padding: 1em;
   }
 
   /* CARDS POPUP */
+  .card-pop {
+    width: 80%;
+  }
```

Comment:
- [ ] **Optional:** Although it isn't a requirement, Kindly adjust the width of your card  `width: 40%;`. And make some adjustments to the card to be more visually appealing.

Reply:
@brytebee Will sure do that, Thanks again 🤞.